File: Java/Easy/PalindromeNumber/Palindrome_Number.java
class Solution {
    public boolean isPalindrome(int num) {
        int a = num;
        int temp = 0;
        while(a > 0)
        {
            int rem = a % 10;
            temp = temp*10 + rem;
            a = a/10;
        }
        return num == temp;
    }
}

File: Python/Easy/Palindrome_Number/Palindrome_Number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        revnum = int(str(x)[::-1])

        if x == revnum:
            print(f"{x} is a Palindrome")
        else:
            print(f"{x} is not a Palindrome")
x = int(input("Enter a number : "))

solution = Solution()

solution.isPalindrome(x)